easybuild/easyconfigs/n/NWChem/NWChem-6.6.revision27746-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,40 @@
+name = 'NWChem'
+version = '6.6.revision27746'
+verdate = '2015-10-20'
+pyver = '2.7.11'
+versionsuffix = '-%s-Python-%s' % (verdate, pyver)
+
+
+homepage = 'http://www.nwchem-sw.org'
+description = """NWChem aims to provide its users with computational chemistry tools that are scalable both in
+their ability to treat large scientific computational chemistry problems efficiently, and in their use of available
+parallel computing resources from high-performance parallel supercomputers to conventional workstation clusters.
+NWChem software can handle: biomolecules, nanostructures, and solid-state; from quantum to classical, and all
+combinations; Gaussian basis functions or plane-waves; scaling from one to thousands of processors; properties
+and relativity."""
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://www.nwchem-sw.org/download.php?f=']
+sources = ['Nwchem-%s-src.%s.tar.gz' % (version, verdate)]
+
+patches = [
+    'NWChem_fix-date.patch',
+    'Ga_argv.patch',
+    'Raman_displ.patch',
+    'Ga_defs.patch',
+    'Cosmo_meminit.patch',
+    'Dplot_tolrho.patch',
+    'Sym_abelian.patch',
+    'Xccvs98.patch',
```

Comment:
@RvDijk it's better to name these `NWChem-6.6.revision27746_foo.patch` to make it clear on top of which NWChem version these were created